monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc

```diff
@@ -322,6 +417,10 @@ Appears in: link:#clustermonitoringconfiguration[ClusterMonitoringConfiguration]
 | Property | Type | Description
 |collectors|link:#nodeexportercollectorconfig[NodeExporterCollectorConfig]|Defines which collectors are enabled and their additional configuration parameters.
 
+|maxProcs|uint32|The target number of CPUs on which the node-exporter's process will run. The default value is `0`, which means that node-exporter runs on all CPUs. If a kernel deadlock occurs or if performance degrades when reading from `sysfs` concurrently, you can change this value to `1`, which limits node-exporter to running on one CPU. For nodes with a high CPU count, you can set the limit to a low number, which  saves resources by preventing Go routines from being scheduled to run on all CPUs. However, I/O performance degrades if the `maxProcs` value is set too low and there are many metrics to collect.
+
+|ignoredNetworkDevices|*[]string|A list of network devices, defined as regular expressions, that you want to exclude from the relevant collector configuration such as `netdev` and `netclass`. If no list is specified, the Cluster Monitoring Operator uses a predefined list of devices to be excluded to minimize the impact on memory usage. If the list is empty, no devices are excluded. If you modify this setting, monitor the `prometheus-k8s` deployment closely for excessive memory usage.
+
```

Comment:
missed `resources` setting for NodeExporterConfig
https://github.com/openshift/cluster-monitoring-operator/blob/release-4.14/pkg/manifests/types.go#L315

Reply:
Fixed